python/mlx/nn/layers/linear.py

```diff
@@ -15,31 +31,98 @@ class Linear(Module):
 
         y = W^\top x + b
 
-    where :math:`W` has shape ``[output_dims, input_dims]``.
+    where:
```

Comment:
Something doesn't add up between the equation and the shape of `W`. I think the equation should read $x W^\top + b$ instead.

Reply:
I've changed it to `y = x W^\top + b`